### PR TITLE
Revert "Integrations Lock - set CI project id"

### PR DESCRIPTION
**Pull request by Itay4**
Reverts demisto/demisto-sdk#1270

### CODE REVIEW

**Review by hod-alpert — APPROVED** [review]

File: demisto_sdk/commands/test_content/IntegrationsLock.py [hunk 1]
```diff
@@ -106,14 +106,20 @@ def workflow_still_running(workflow_id: str, test_playbook) -> bool:
                                                      auth=(CIRCLE_STATUS_TOKEN, ''))
             workflow_details_response.raise_for_status()
         except Exception:
+            if os.getenv('CIRCLECI'):
+                test_playbook.build_context.logging_module.warning(
+                    f'Failed to check status for circle workflow with id {workflow_id}, '
+                    'assuming it\'s a working gitlab pipeline')
+                return True
```
Comment (yaakovi) [inline demisto_sdk/commands/test_content/IntegrationsLock.py:113]: why did you add this part?

Comment (hod-alpert) [reply demisto_sdk/commands/test_content/IntegrationsLock.py:113]: Because circle builds cannot actually check for gitlab builds status
